modules/beetmover_scriptworker/manifests/settings.pp

```diff
@@ -148,24 +150,21 @@
             verify_cot_signature                    => true,
             cot_product                             => 'thunderbird',
         },
-        'mobile-staging' => {
-            # TODO: once bug 1490381 is fixed, we should change these to more restrictive counterparts
-            dep_maven_beetmover_aws_access_key_id => secret('dep_maven_beetmover_aws_access_key_id'),
-            dep_maven_beetmover_aws_secret_access_key => secret('dep_maven_beetmover_aws_secret_access_key'),
-            dep_maven_buckets => {
-                geckoview   => 'maven-default-s3-bucket-wj4w05pguw64',
+        'mobile-prod' => {
+            prod_maven_beetmover_aws_access_key_id => secret('prod_maven_android_components_beetmover_aws_access_key_id'),
+            prod_maven_beetmover_aws_secret_access_key => secret('prod_maven_android_components_beetmover_aws_secret_access_key'),
+            prod_maven_buckets => {
+                components   => 'maven-default-s3-bucket-1svmqy68t6xd4',
             },
 
             worker_type                             => 'mobile-beetmover-v1',
             worker_group                            => 'mobile-beetmover-v1',
             taskcluster_client_id                   => 'project/mobile/android-components/releng/scriptworker/beetmover/production',
             taskcluster_access_token                => secret('beetmoverworker_prod_taskcluster_access_token_mobile'),
-            taskcluster_scope_prefix                => 'project:mobile:android-components:releng',
-            # TODO to turn this back on
-            sign_chain_of_trust                     => false,
+            taskcluster_scope_prefix                => 'project:mobile:android-components:releng:beetmover:',
+            sign_chain_of_trust                     => true,
             verify_chain_of_trust                   => true,
-            # TODO to turn this back on
-            verify_cot_signature                    => false,
+            verify_cot_signature                    => true,
```

Comment:
Even with these turned on, things should work smoothly, because we don't yet have the `json-e` right? For some reason I'm still worried a bit as I haven't added cot-gpg keys for this particular instance and I'm unsure if this will break it or not.

Reply:
That's right, it will. json-e is skipped at this stage https://github.com/mozilla-releng/scriptworker/blob/d7ac2ee55e7fb537b939090358692ebd2131b725/scriptworker/cot/verify.py#L1218.

There is no need to create a new cot-gpg key these days. We only rely on a single key https://github.com/mozilla-releng/cot-gpg-keys/tree/a53350f58c91d3bd9845bf6ff123f588ec2e43c4/scriptworker/valid

Reply:
👍